mak/DOCS

```diff
@@ -1,5 +1,8 @@
 DOCS=\
 	$(DOCDIR)\object.html \
+    \
+    $(DOCDIR)\object__lifetime.html \
+    \
```

Comment:
Spaces where should be tabs?

Reply:
I'm not sure about this location, but make can be fussy about tabs vs spaces.

Reply:
Done